azavea/tilegarden
# Feature/remove nginx container

## Overview

After the binary content issues with claudia-local-api were fixed, it turned out that using nginx in development to mimic the behavior of the production CloudFront distribution was unnecessary. claudia-local-api uses express under the hood, which lacks API Gateway's binary content serving issues. This PR removes the nginx docker container from the project and switches the port of the development server from 9001 to 3000, to cut down on confusion from the message that the server prints on startup.

## Testing Instructions

 * Start the development server with `./scripts/server`.
 * Navigate to http://localhost:3000. The expected landing page should be there, instead of at port 9001.


Resolves #81
